webapp/app/templates/lotse/display_filing_failure.html

```diff
@@ -1,12 +1,18 @@
-{% extends 'basis/base_step_display.html' %}
+{% extends 'basis/display_failure_icon.html' %}
 
-{% block display_content %}
-    <div class="alert alert-danger" role="alert">
-        <h1 class="alert-heading">{{ _('form.lotse.filing.failure.alert.title') }}</h1>
-        {% for error in elster_data['validation_problems'] %}
-            <ul>
-                <li>{{ error }}</li>
-            </ul>
-        {% endfor %}
+{% set title = _('form.lotse.filing.failure.alert.title') %}
+{% set error_details = elster_data['validation_problems'] %}
```

Comment:
Okay, to be honest this confused me a little. Because this is not set in the `display_failure_icon` I was a little confused. I then overlooked that it is set in `display_failure`. 

Maybe we either should name it a little bit more explicit here (`validation_problem`?) or we do set it in the step.

Reply:
I we'd "name it a little bit more explicit here (validation_problem?)" here, we'd need to also rename it in `eligibility/display_failure.hmtl` and I don't think that's a good idea, that's why I chose the more general name of "error_details". But yes, we can set it in the step's `render` method if you prefer that. I'll then also set it in the eligibility failure steps, instead of in `eligibility/display_failure.hmtl`.

Reply:
Let's maybe quickly talk about it

Reply:
We've decided to rename it to `eligibility/display_failure_with_icon.html`